Class-9-(Unit-Testing-&-TDD).md

```diff
@@ -0,0 +1,17 @@
+## In the prep work this week, the students learned:
+1. What unit testing is and why it is important.
+1. How to use ``Jasmine`` for testing.
+
+## Announcements
+1. More info to come... when in doubt, check with the class Candidate Engagement Manager!
+1. When your third graded assignment becomes available, make sure to check it out! Starting as soon as possible is key to finishing those graded assignments on time!
+
+## Large Group Time (Instructor)
+
+### Class 9 Topics That Require Careful Attention
+1. Reinforce why unit testing is important. If you can share an instance where properly testing your code saved you a huge headache, please do! 
+
+## Studio (TA Notes)
+
+### Class 9 Studio
```

Comment:
Other items:
a) Reinforce the benefits of writing tests before the code.
b) If many of the students have the same questions or problems with writing their first test, consider going through that as a group.
c) This would be a good studio for the students to work on in pairs. Each student still needs to submit their own final project, but collaborating on the code helps with the learning curve.

Reply:
Done! Thanks!